### DISCUSSION TITLE
How to visualize convolutional filter outputs

### DISCUSSION THREAD

**Discussion by al3ms** [Project-MONAI/MONAI#7001] [General]
I trained my model using the great framework MONAI, which really simplified the implementation.

I am wondering if MONAI supports visualizing the convolutional filter outputs easily, so I could understands what are the learned features.

Thanks

**Comment by KumoLiu** [comment]
Hi @al3ms, you can refer to these tutorials.
https://github.com/Project-MONAI/tutorials/tree/main/modules/interpretability
Thanks!

**Comment by wyli** [comment]
also it's possible to use this forward hook approach https://web.stanford.edu/~nanbhas/blog/forward-hooks-pytorch/#forward-hooks-101 to capture the activations.

and once the activation maps are stored, `monai.visualize.matshow3d` can display N-d tensors as a set of 2D frames
